pkg/compactor/job.go

```diff
@@ -174,11 +173,9 @@ func jobWaitPeriodElapsed(ctx context.Context, job *Job, waitPeriod time.Duratio
 			continue
 		}
 
-		metaPath := path.Join(meta.ULID.String(), block.MetaFilename)
-
-		attrs, err := userBucket.Attributes(ctx, metaPath)
+		attrs, err := block.GetAttributes(ctx, meta, userBucket)
```

Comment:
`meta.LastModified` may already be set when we get here, why not use it?

Reply:
Sure